ui/js/controllers/perf/alerts.js

```diff
@@ -423,6 +423,10 @@ perf.controller('AlertsCtrl', [
                                 resultSet.dateStr = dateFilter(
                                     resultSet.push_timestamp*1000, thDateFormat);
                                 // want at least 14 days worth of results for relative comparisons
+                                if (repo === "mozilla-beta") {
+                                    // default to 90 days for beta branch
+                                    phDefaultTimeRangeValue = 7776000;
+                                }
```

Comment:
You're redefining what's supposed to be a constant here (phDefaultTimeRangeValue), which is kind of confusing. I think it would be better to make a map of overrides to this value in values.js and then use that here.

E.g. in values.js:

```js
treeherder.value("phTimeRangeValues", {
    "mozilla-beta": 7776000
});
```

Then you can just look that up in here like this:

```js
let timeRange = phTimeRangeValue[repo] ? phTimeRangeValue[repo] : phDefaultTimeRangeValue
```

Make sense?